docker/Dockerfile

```diff
@@ -1,15 +1,14 @@
 FROM jupyter/datascience-notebook:latest
-MAINTAINER Serenata de Amor "datasciencebr@gmail.com"
 
 USER root
 
-RUN apt-get update && apt-get install -y \
-  unzip
+RUN apt-get update && apt-get install -y unzip
 
 USER jovyan
 
 COPY requirements.txt ./
 COPY conda_requirements.txt ./
+COPY config.ini.example ./config.ini
```

Comment:
Hum... have you guys discussed about just setting them up as `ENV`s on the `Dockerfile`?  (obviously, don't add any secret key there 😅) That way, people just willing to try things out would be a `git clone ... && docker-compose up -d` from getting the env up and running while allowing env vars to be overwritten by adding some entries under the `environment` config of `docker-compose.yml` or just editing the `config.ini`.

But I believe we'll need to change the code that makes use of those settings to prioritize env vars and fallback to the config file right?